compiler-core/templates/docs-css/index.css

```diff
@@ -439,8 +440,8 @@ p code {
   font-size: calc(0.8 * var(--sidebar-toggle-size));
 }
 
-body.drawer-closed .label-open,
-body.drawer-open .label-closed {
+html[data-drawer="closed"] .label-opened,
+html[data-drawer="opened"] .label-closed {
```

Comment:
Why has this changed?

Reply:
the data attrib lives on the html elem and is easily toggable and available very early versus classes on body which are available after `<head>` not before.